src/renderer/components/QueryEditor/QueryEditor.tsx

```diff
@@ -66,6 +66,7 @@ export default class QueryEditor extends React.Component<Props> {
         <span>
           <i className="fas fa-check" />
         </span>
+        <span>execute: {query.runAt?.format("MM/DD HH:mm:ss") ?? "-"}</span>
```

Comment:
I think it is better to include the year.

Reply:
YY? YYYY?

Reply:
I prefer YYYY.